### ISSUE TITLE
Hardened doesn't have language based archives

### ISSUE THREAD

**Issue by dugwood** [micahflee/torbrowser-launcher#210]
Just tried your application today, and it tries to download https://dist.torproject.org/torbrowser/5.5a4-hardened/tor-browser-linux64-5.5a4-hardened_fr.tar.xz which is not available. Only https://dist.torproject.org/torbrowser/5.5a4-hardened/tor-browser-linux64-5.5a4-hardened_ALL.tar.xz is available.

It breaks afterwards (I've forced ALL as the locale), because the archive hasn't got the «ALL» in the subdirectory (/tbb/x86_64/tor-browser_ALL/start-tor-browser doesn't exist, /tbb/x86_64/tor-browser/start-tor-browser does).

As stated in another issue, this would be great to select the current status, or at least to fix the language list in case it's an hardened archive.

Maybe this has changed over the time, I'm not familiar with the project.

I'm running Debian 8 (Jessie), up to date.

Thanks for your work.

**Comment by FuzzyToothpaste** [comment]
I'm having this same problem on Debian Testing. This issue probably breaks Tor Browser Launcher for most users on most systems, and it would be nice if we could choose between channels.

**Comment by nickpellant** [comment]
I'm experiencing the same problem on Debian Jessie (backports). torbrowser-launcher tries to pull the hardened image using en-US, however hardened images are only provided with an ALL lang.

**Comment by jmcastellote** [comment]
Dear micahflee, first things first - huge thank you for your brilliant work.
Same issue here with Debian Jessie.
Tried to use ppa repo but I think this works only in Ubuntu.
According to micahflee, torbrowser-launcher gets confused with the actual latest release, and latest version of launcher does not have this issue. However latest version is not yet available at Jessie backports.
Any suggestion to work it around?

**Comment by micahflee** [comment]
Sorry about this! Yeah, TBL isn't supposed to download the "hardened" version at all, but rather the latest stable. I've fixes this in version 0.2.2, but until that version is available in the repos then I suggest you delete your TBL files and build from source.

In short, delete your old TBL files:

```sh
rm -r .cache/torbrowser .config/torbrowser .local/share/torbrowser
```

Install dependencies and build from source:

```sh
sudo apt-get install build-essential dh-python python-all python-stdeb python-gtk2 python-psutil python-twisted python-lzma python-txsocksx gnupg fakeroot xz-utils tor
git clone https://github.com/micahflee/torbrowser-launcher.git
cd torbrowser-launcher
./build_deb.sh
sudo dpkg -i deb_dist/torbrowser-launcher_*.deb
```

**Comment by jmcastellote** [comment]
Hi micahflee,

The workaround works like a charm. Many thanks!

Cheers
